src/Stand.jsx

```diff
@@ -213,6 +217,13 @@ const Stand = () => {
                     </div>
                 </div>
             </div>
+            <div>
+                <Button xType={'secondary'}
+                        onClick={() => setIsOpen(true)}
+                        title={'Open popover'}
+                        size={'block'}/>
+                {isOpen && <Popover setIsOpen={setIsOpen}/>}
```

Comment:
your dialog should know itself when it is opened. so this line should look like this
```suggestion
                <Popover isOpen={isOpen} setIsOpen={setIsOpen}/>
```
when `isOpen` is true, then the dialog is visible